# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1290706303 18000
#      Thu Nov 25 12:31:43 2010 -0500
# Node ID 2bbea68c8abc0913d9cec86adfcb2e2593ef13e9
# Parent  75ff3db6a6872c5990c8191b5fe0d9fe75f512bb
ChangeLog fix for EXEC_PATH changes accidentally checked in with datetick change in e2a4f3478b7c

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,15 @@
+2010-11-25  John W. Eaton  <jwe@octave.org>
+
+	* defaults.cc (set_exec_path): Append EXEC_PATH to PATH.  Store
+	only extra directories in EXEC_PATH.
+	(FEXEC_PATH): Update doc string.
+	Bug #31695.
+
 2010-11-22  John W. Eaton  <jwe@octave.org>
 
 	* defun.cc (defun_isargout): Cast nout to octave_idx_type in call
 	to std::min.
 
 2010-11-21  Kai Habel  <kai.habel@gmx.de>
 
 	* DLD-FUNCTIONS/fltk-backend.cc (__fltk_uigetfile__): New function
